tests/lit-tests/1739.ispc

```diff
@@ -0,0 +1,4 @@
+//; RUN: not %{ispc} --target=avx2-i32x8 --target=host %s --nowrap 2>&1 | FileCheck %s
```

Comment:
Only one ``--target=...`` is needed here.

Reply:
Done